openapi.yaml

```diff
@@ -5359,7 +5997,10 @@ components:
           nullable: true
           default: false
         suffix:
-          description: The suffix that comes after a completion of inserted text.
+          description: |
+            The suffix that comes after a completion of inserted text.
+
+            This parameter is only supported for `gpt-3.5-turbo-instruct`.
```

Comment:
This modified description is missing from the TypeSpec. It applies to `CreateCompletionRequest` in completions/models.tsp.